libqfieldsync/project_checker.py

```diff
@@ -410,3 +415,22 @@ def check_layer_package_prevention(
             return FeedbackResult(main_msg)
 
         return None
+
+    def check_external_layers(
+        self, layer_source: LayerSource
+    ) -> Optional[FeedbackResult]:
+        home_path = Path(self.project.fileName()).parent
+
+        if layer_source.is_file and not layer_source.is_localized_path:
+            layer_path = Path(layer_source.layer.source()).resolve()
```

Comment:
This seems to be causing some hiccups, see https://github.com/opengisch/qfieldsync/issues/630?notification_referrer_id=NT_kwDOAAj6d7IxMzc0MTMwNDc4Mzo1ODg0MDc#issuecomment-2525185190

The source is more than just a path

Reply:
My bad I overlooked this during the review. Thanks for the pointer!